modules/acs-requirements.adoc

```diff
@@ -28,12 +28,11 @@ For more information, see link:https://access.redhat.com/node/5822721[{product-t
 
 Cluster nodes minimum requirements:
 
-* *Architecture*: amd64, ppc64le, or s390x
+* *Architecture*: `amd64`, `ppc64le`, or `s390x`
 +
 [NOTE]
 ====
-For ppc64le, or s390x architectures, you can only install {product-title-short} secured cluster services on {ibm-power}, {ibm-z}, and {ibm-linuxone} clusters.
-Central is not supported at this time.
+For `ppc64le`, or `s390x` architectures, you can only install {product-title-short} secured cluster services on {ibm-power}, {ibm-z}, and {ibm-linuxone} clusters.
```

Comment:
@SNiemann15 both Secured Cluster services and Central services could be installed now, not only secured cluster services.